Tests/PackageLoading/ConventionTests.swift

```diff
@@ -159,21 +158,42 @@ private extension FileSystem {
     }
 }
 
-/// Loads a package using PackageBuilder at the given path. If package description is provided it is used,
-/// otherwise a default package with name of directory is passed to manifest.
+/// A convenience enum for package builder tester. Since most of our convention tests
+/// only requires a Package object with the name filled in.
+enum PackageOrName {
```

Comment:
How often do we use the variant when a `Package` is provided? Maybe we should do that using another mechanism (we could just do it by including an appropriate `Package.swift` in the FS)?